AdSecGHTests/Helpers/DataConvertorTest.cs

```diff
@@ -68,19 +68,25 @@ public void ShouldComputeAndAssignOutputs() {
       Assert.NotNull(output.Value);
     }
 
+    [Fact]
+    public void ShouldComputeTheRightResult() {
```

Comment:
also add one for unhappy path

Reply:
hmm can we think about having conversion tests instead? This suite of tests is testing the default Parameters on a fake component. I've added a test, but it is a bit contrived.